tests/test_split_segments_cmfd/results_true.dat

```diff
@@ -0,0 +1,60 @@
+# Iterations: 25
+keff:  1.04701E+00
```

Comment:
Ditto, doesn't seem like the fluxes and eigenvalue are needed in the results for this test.